packages/blockchain-wallet-v4/src/network/api/custodial/index.ts

```diff
@@ -38,9 +39,17 @@ export default ({ authorizedGet, authorizedPost, nabuUrl }) => {
       }
     })
 
+  const getWithdrawalFees = (): WithdrawalMinsAndFeesResponse =>
+    authorizedGet({
+      url: nabuUrl,
+      ignoreQueryParams: true,
+      endPoint: '/payments/withdrawals/fees?product=simplebuy'
```

Comment:
this will not work because it's still not open for simplebuy, if you want to test this please replace it with `payments/withdrawals/fees?product=mercury`

Reply:
will the product param ever need to change for this?  if so, maybe we pass in the product key

Reply:
it might be, can move it to some more generic place because we use similar call on other place where we do load only `minAmounts` not fees.